digitalout.py

```diff
@@ -50,11 +49,14 @@ def load_xml(self, node):
                     self.clockRate = float(child.text) 
                     
                 elif child.tag == "startTrigger":
+
+                    # This is modifying the definitions of the outer for loop child and node. This
```

Comment:
I had thought that the iterated variable would be passed into scope but that seems to not be the case. Interesting. The child variable is still getting overwritten at scope but because of the elif structure of the function that doesn't affect the outcome. I think it would still be best to rename for the sake of clarity when reading so the reader doesn't have to be mindful of that scope gap.